styles/pinterest/catppuccin.user.css

```diff
@@ -193,12 +195,24 @@
       color: var(--color-text-inverse);
     }
 
+    // Profile button
+    // Invite collaborators buttons
+    div[style="border: 1px solid white;"],
     div[data-test-id="gestalt-avatar-svg"] {
-      box-shadow: 0 0 0 1px @crust !important;
+      box-shadow: 0 0 0 0 @surface1 !important;
+      outline-color: @base !important;
+      outline-width: 0 !important;
+      border-color: @base !important;
+      border-width: 1px !important;
```

Comment:
do we need this line?

Reply:
border-color should be base, but we don't need to set border-width (and it should not be zero). We could either set outline-color to base or outline-width to zero, but we don't need both. Setting outline-color to base seems more consistent since it sets border-color as well.

Reply:
It also doesn't look like we need to set background-color right here either, and setting text to subtext1 isn't consistent with the color of other icons I believe